### DISCUSSION TITLE
Any plans to merge in CloudFlare's PR?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [pgbouncer/pgbouncer#780] [General]
https://github.com/cloudflare/cf-pgbouncer/pull/1 - as it has a few important bug fixes it would be great to see them merged back into mainline.

**Comment by petere** [comment]
The normal route is that pull requests or issue reports are submitted to this project. I don't think we are going to go around digging proactively through other people's forks in case there is anything interesting in there to merge.
